cmd/sync.go

```diff
@@ -187,7 +191,9 @@ type cookedSyncCmdArgs struct {
 	recursive             bool
 	followSymlinks        bool
 	include               []string
+  includePath           []string
```

Comment:
minor: turn on file watcher to auto-format when you save

Reply:
Gah! How did that make it through? I've had it on for ages.
![image](https://user-images.githubusercontent.com/49764384/62387451-7abf3900-b50f-11e9-853c-6295b5346ca9.png)

Reply:
hmmm that's strange indeed, maybe they have a bug. Hope an upgrade could fix it.

Reply:
I've had the same thing occasionally, where i'v had it turned on but it didn't process some things.

Reply:
Yeah, sometimes it'll process in the middle of my work and I'll have to control+Z a bit to get it back